src/ostree/ot-builtin-refs.c

```diff
@@ -245,7 +247,18 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
 
           if (!ostree_parse_refspec (refspec, &remote, &ref, error))
             goto out;
-          
+
+          /* Look for alias if it exists for a ref we want to delete */
+          GLNX_HASH_TABLE_FOREACH_KV (ref_aliases, const char *,
+                                      ref_alias, const char *, value)
+            {
+              if (!strcmp(ref, value))
+              {
+                g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED,
+                       "Ref %s has some aliases, first delete corresponding aliases", ref);
```

Comment:
How about: `"Ref '%s' has an active alias: '%s'", ref, ref_alias`

Reply:
Makes sense, updated the error message.